.rubocop.yml

```diff
@@ -30,7 +30,7 @@ Style/Documentation:
     - 'spec/**/*.rb'
 
 Style/StringLiterals:
-  EnforcedStyle: double_quotes
+  EnforcedStyle: single_quotes
```

Comment:
why?)

Reply:
Oh, I just love enforcing style to single_quotes. I'll change that back :)